test_proxy_camoufox.py

```
@@ -0,0 +1,35 @@
+    }
+    
+    logger.info("Testing Camoufox with proxy...")
+    
+    try:
+        with Camoufox(headless=True, geoip=True, proxy=proxy_config) as browser:
+            page = browser.new_page()
+            
+            logger.info("Navigating to ipv4.icanhazip.com...")
+            page.goto("https://ipv4.icanhazip.com")
+            
+            content = page.content()
```

Ruff (0.14.7)

Local variable `content` is assigned to but never used

Remove assignment to unused variable `content`

(F841)